rollup.config.js

```diff
@@ -10,6 +10,24 @@ import { dependencies } from './package-lock.json'
 import fs from 'fs'
 import path from 'path'
 
+import * as edge from 'edge.js'
+edge.registerViews(path.join(__dirname, './packages/shell-chrome/views'))
+const renderPanel = () => {
+    let panel = edge.render('master')
```

Comment:
Could we rename the template file to `panel.edge` as well?

Reply:
Ah yeah of course, sorry is that what you were saying before?

Reply:
Haha yes, but the variables too 😂😂

Reply:
😆